packages/nimble-components/docs/component-usage-patterns/combobox-select-menu.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story } from '@storybook/blocks';
+import * as selectStories from '../../src/select/tests/select.stories';
+import * as comboboxStories from '../../src/combobox/tests/combobox.stories';
+import * as menuButtonStories from '../../src/menu-button/tests/menu-button.stories';
+
+<Meta title="Usage/Combobox, Select, & Menu Button"/>
+
+# Combobox, Select, & Menu Button Usage
+
+### What's the same?
+The `nimble-select`, `nimble-combobox`, and `nimble-menu-button` are all Nimble components used for presenting a list 
+of options to the user. 
+
+### What's the difference?
```

Comment:
When providing contrast, we should aim to use the same sentence structure. Use simple but specific language and describe the front-end interaction as it relates to the user. For example:

- Use `nimble-select` for a list of predefined options that can be selected by the user
- Use `nimble-combobox` for a list of predefined options that can be searched and selected by the user
- Use `nimble-menu-button` for a list of navigation and action commands that can be triggered by the user